### ISSUE TITLE
Paginated grid results summary do not meet colour contrast

### ISSUE THREAD

**Issue by mmafe** [ccswbs/ugnext#374]
Paginated grid results summary do not meet colour contrast. Recommend using one of the U of G Tailwind classes to fix (likely `--uog-color-body-copy`

Affects multiple pages using the paginated grid, such as:
- https://www.uoguelph.ca/csahs/people
- https://www.uoguelph.ca/ovc/news

<img width="1707" height="892" alt="Image" src="https://github.com/user-attachments/assets/3509466d-c70b-422b-ae99-02b6caabc608" />

---

Title: WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.mb-4)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: College of Social and Applied Human Sciences People | University of Guelph - https://tq-next.netlify.app/csahs/people

Element path: .mb-4

Snippet: <div class="mb-4 text-center text-sm opacity-70">Showing 20 of 285 results</div>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 3.54 (foreground color: #888888, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 147.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
